<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite name="TestAddBusinessOwner-20250602153606" tests="1" file=".py" time="40.474" timestamp="2025-06-02T15:36:46" failures="0" errors="1" skipped="0">
		<testcase classname="TestAddBusinessOwner" name="test_add_business_owner" time="40.474" timestamp="2025-06-02T15:36:46" file="nm.py" line="10">
			<error type="TimeoutError" message="Locator.wait_for: Timeout 30000ms exceeded.&#10;Call log:&#10;  - waiting for locator(&quot;div[role='option']:has-text('Male')&quot;).first to be visible&#10;    63 × locator resolved to hidden &lt;div role=&quot;option&quot; aria-label=&quot;Male&quot; id=&quot;gender_list_0&quot; aria-selected=&quot;false&quot;&gt;male&lt;/div&gt;&#10;"><![CDATA[Traceback (most recent call last):
  File "C:\Users\Ekanta Banik Durjoy\Desh\nm.py", line 36, in test_add_business_owner
    page.locator("div[role='option']:has-text('Male')").first.wait_for(state="visible")
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\sync_api\_generated.py", line 17920, in wait_for
    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\_impl\_sync_base.py", line 115, in _sync
    return task.result()
           ~~~~~~~~~~~^^
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\_impl\_locator.py", line 685, in wait_for
    await self._frame.wait_for_selector(
        self._selector, strict=True, timeout=timeout, state=state
    )
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\_impl\_frame.py", line 323, in wait_for_selector
    await self._channel.send("waitForSelector", locals_to_params(locals()))
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\_impl\_connection.py", line 61, in send
    return await self._connection.wrap_api_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
    )
    ^
  File "C:\Users\Ekanta Banik Durjoy\AppData\Local\Programs\Python\Python313\Lib\site-packages\playwright\_impl\_connection.py", line 528, in wrap_api_call
    raise rewrite_error(error, f"{parsed_st['apiName']}: {error}") from None
playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 30000ms exceeded.
Call log:
  - waiting for locator("div[role='option']:has-text('Male')").first to be visible
    63 × locator resolved to hidden <div role="option" aria-label="Male" id="gender_list_0" aria-selected="false">male</div>

]]></error>
			<system-out><![CDATA[Clicking 'Add Business Owner' button...
Modal appeared!
]]></system-out>
		</testcase>
	</testsuite>
</testsuites>
